lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -464,6 +471,8 @@ def add_merchant_data(xml, options)
         xml.tag! 'clientLibrary', 'Ruby Active Merchant'
         xml.tag! 'clientLibraryVersion', VERSION
         xml.tag! 'clientEnvironment', RUBY_PLATFORM
+        xml.tag!('partnerSolutionID', application_id) if application_id
```

Comment:
what is the `application_id`?

Reply:
fyi https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateway.rb#L141

Reply:
thank you!